templates/method.mustache

```diff
@@ -152,19 +146,13 @@
 
 {{/parameters}}
 
-if(parameters.$queryParameters) {
-    Object.keys(parameters.$queryParameters).forEach(function(parameterName){
-        queryParameters[parameterName] = parameters.$queryParameters[parameterName];
-    });
-}
-
     {{^isBodyParameter}}
         {{#isPOST}}
             form = queryParameters;
             queryParameters = {};
```

Comment:
I just noticed that you are explicitly nuking any `query` params here for `POST` requests... why?
this might interfere with my change of moving the `query params` handling into the `request()` method...

Reply:
Good question! This came from the old fork of the project, so I am not entirely sure. It is ofc. allowed to send querystring params with POST requests, so I can't think of an obvious reason at least. @Markionium, @scottc - Thoughts?

Reply:
I generally agree with the proposed changes.

My preference is to have backwards compatibility and use immutable data structures everywhere possible.

If it doesn't break any of the unit tests. That should suffice for backwards comparability purposes imo.

Anyone using custom templates, could incorporate the changes at their own leisure.